src/reference/asciidoc/amqp.adoc

```diff
@@ -32,7 +32,7 @@ It provides much more in-depth information regarding Spring's integration with A
 
 A configuration sample for an AMQP Inbound Channel Adapter is shown below.
 
-[source,xml]
+[source]
```

Comment:
This needs to be reverted - my editor was having problems with asciidoc syntax highlighting and I thought it was because of the `<n>` tags (bad xml). But it turned out to be due to the unmatched `*` in some descriptions (see back tick changes around some text below) and I failed to restore this line.